crates/pallet-humanode-session/src/migrations/v1.rs

```diff
@@ -80,6 +87,11 @@ pub fn pre_migrate<T: Config>() -> Vec<u8> {
 /// Panics if anything goes wrong.
 #[cfg(feature = "try-runtime")]
 pub fn post_migrate<T: Config>(state: Vec<u8>) {
+    // Empty state means that the check is disabled.
+    if state.is_empty() {
```

Comment:
Isn't `0u64.encode().is_empty() == true` as well?

Reply:
Will try

Reply:
Tried `return 0u64.encode();` and got failed assertion.

Reply:
So it's non-empty? Well then